StonehearthEditor/Recipes/RecipesView.Designer.cs

```diff
@@ -38,8 +38,9 @@ private void InitializeComponent()
             this.toolStrip = new System.Windows.Forms.ToolStrip();
```

Comment:
Is it easy to make the rows unresizable? Half the time I'm trying to click a row, it's a no-op because the cursor is a resize arrow.

Reply:
Apparently just setting AllowUserToResizeRows to false isn't enough, I have to set autoresizerow to something other than none, which makes the row cells have a bigger height. Now it should be preventing users to resize and hopefully the larger height isn't a problem.